pkg/intel/metadata/manifest/common/manifestcodegen/cmd/manifestcodegen/main.go

```diff
@@ -76,6 +76,10 @@ func processPath(path string, isCheck, enableTracing bool) error {
 		}
 	}
 
+	// ugly terrible hack to workaround versioning support
+	// TODO: fix the importer to recognize `/v2/` as version, not as path
+	os.Setenv("GO111MODULE", "off")
```

Comment:
Having messed enough with Go modules, this is my favourite part, haha :D

Reply:
Oh I see, it was removed, and I brought it back when attemping to resolve the conflicts... sorry, needs to be undone.